modules/rostests/apitests/compiler/ms/seh/seh0001.c

```diff
@@ -6,7 +6,6 @@
 #include "seh.h"
 
 char test[] = "SEH0001.c";
-int perfect;
```

Comment:
I wonder what was their idea behind this.
Do you think it's possible to send that upstream?

Reply:
No, I have already spent enough time on this :P